### ISSUE TITLE
[Question] Install latest working version

### ISSUE THREAD

**Issue by chalcosoma** [pragmagic/godot-nim#43]
Hello! I'm trying to use Godot with Nim but the latest version is not working. I tried to use an older version (0.7.8) but I don't know how to install it correctly. I downloaded the zip file and extracted it to /home/user/.nimble/pkgs because I don't know how to install it using nimble directly. When I tried to compile the "godot-nim-stub" I got an error saying "nakefile.nim(5, 8) Error: cannot open 'godotapigen'". How do I install the latest working version?

Thanks in advance! =)

**Comment by chalcosoma** [comment]
I'm using choosenim with version 0.18.0. I tried to install with "nimble install godot" but when I compile the stub project with "nake build" I got the following error: 

```
/home/user/.nimble/pkgs/godot-0.7.17/godotapigen.nim(9,` 26) Error: type mismatch: got <string>
but expected one of: 
proc getIdent(ic: IdentCache; identifier: string): PIdent
proc getIdent(ic: IdentCache; identifier: string; h: Hash): PIdent
proc getIdent(ic: IdentCache; identifier: cstring; length: int; h: Hash): PIdent

expression: getIdent(ident)
```
I saw it is the same error as https://github.com/pragmagic/godot-nim/issues/41 but I couldn't find a solution, then I tried to use godot-nim 0.7.8 that gives me this other error:

`nakefile.nim(5, 8) Error: cannot open 'godotapigen'`

My .bash_profile has this as GODOT_BIN:

`export GODOT_BIN=$HOME/Godot3/Godot_v3.0.6-stable_x11.64`

**Comment by zetashift** [comment]
Yeah, use nimble and you can solve that `getIdent` error with doing some trickey explained here: https://github.com/pragmagic/godot-nim/issues/32#issuecomment-399938041

after that `nake clean` and `nake build` in the godot-nim-stub

**Comment by zetashift** [comment]
The nimble install thing is a weird thing that just seemed to work

Alright let's see if I can reproduce my solution
-> Clean install with choosenim
-> `nimble install godot`
-> `nimble install nake`

Now if I run `nake clean` I actually get the Pident error. I went to .choosenim\toolchains\nim-0.18.0 and did `nimble install`  then went to the godot-nim-stub and ran `nake clean` and `nake build` succesfully.
You did the same thing but installed godot-nim after the nimble install, maybe try to install it before hand?

**Comment by zetashift** [comment]
@chalcosoma I think you can close this sicne this issue will mostly likely be solved by 0.19+ compatibility.
